server/lib/resolverMaker.js

```diff
@@ -33,9 +54,29 @@ module.exports = function (dataSourceTypes, dataSourceClients, resolverMappingsJ
 }
 
 function buildPostgresResolver (dataSourceClient, requestMapping, responseMapping) {
+  // use Handlebars.precompile to fail early during initialization
+  try {
+    Handlebars.precompile(requestMapping)
+  } catch (ex) {
+    console.error('Compilation error in requestMapping: ' + requestMapping)
+    console.error(ex)
+    throw new Error('Compilation error in requestMapping: ' + requestMapping)
```

Comment:
@aliok Great to have this.
I'd say the exception thrown by precompile should make its way back to the developer (for the Admin UI).
I'm not sure how best to do that yet, but something to think about.

Reply:
Good point. Let's keep that in mind.